files/preseeds/Centos.kickstart

```diff
@@ -3,7 +3,8 @@
 
 # Use network installation
 url --url="http://mirror.centos.org/altarch/7/os/aarch64/"
-repo --name="ERP-RPM:staging" --baseurl=http://obs.linaro.org/ERP-RPM:/staging/staging/
+#repo --name="ERP-RPM:staging" --baseurl=http://obs.linaro.org/ERP-RPM:/staging/staging/
+repo --name="upstream" --baseurl=http://mirrors.coreix.net/centos-altarch/
```

Comment:
Why not use the standard AltArch? If that works for both ERP and non-ERP, we should just remove the ERP overlay (not comment out)

Reply:
Which one is the standard ?
removed

Reply:
http://mirror.centos.org/altarch/7/os/aarch64/